### PR TITLE
MVP-20341: Implement POST File Rename API for Propel Files to O365 Files

### PR DESCRIPTION
**Pull request by brian-tjoe-walsh**
CloudFileStorage changes made for this task

### CODE REVIEW

File: src/v1/utils/JsForce.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import jsConnect from 'jsforce';
 
 import { logSuccessResponse, logErrorResponse } from '../utils/Logger';
 import InstanceManager from '../utils/InstanceManager';
-import MessageEmitter from '../utils/MessageEmitter';
+// import MessageEmitter from '../utils/MessageEmitter';
```
Comment (mfullmore) [inline src/v1/utils/JsForce.ts:7]: just remove this, no need to comment out imports you are not using.

**Review by mfullmore — APPROVED** [review]

**Comment by clawyq** [comment]
do not merge this , @brian-tjoe-walsh  other than the error handling we spoke about on the call, are there other changes made? I can help put a PR out to get all the relevant changes in

**Comment by brian-tjoe-walsh** [comment]
> do not merge this , @brian-tjoe-walsh other than the error handling we spoke about on the call, are there other changes made? I can help put a PR out to get all the relevant changes in

@clawyq  I don't believe so :o

**Comment by clawyq** [comment]
@brian-tjoe-walsh alright, then the series of PRs should address all the changes required:
https://github.com/PropelPLM/CloudFileStorage/pull/6 and https://github.com/PropelPLM/PLM/pull/10514 (PLM).
Will close this out now.

**Pull request closed without merging**